README.md

```diff
@@ -28,9 +28,6 @@ This gives us the following advantages:
 - opens up for experiment of building the libc from sources;
 - simplify using musl-libc instead of glibc.
 
-Drawbacks:
```

Comment:
It seems `check_cxx_source_compiles` works because we have specified `set (CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)`

At least all the checks in https://github.com/ClickHouse/ClickHouse/blob/master/cmake/cpu_features.cmake#L37-L165 work as expected when doing pseudo cross compiling.

Reply:
We aim to remove all the checks in cmake, so it will never run compiler or linker to figure out configuration.